k8s-tf-cluster/image/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.9 as installer
```

Comment:
A big part of me wishes that we build deterministic Docker images with Bazel, since all we're doing is extracting a known zip and just setting some ENV vars.

But I'm fine with a Dockerfiles for now.